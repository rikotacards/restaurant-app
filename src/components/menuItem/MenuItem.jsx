import React from "react";
import './menu-item.css';

export const MenuItem = props => {
    //To do, pass in props
  return (
   <div className='container'>
       <div className='title'>Pad Thai</div>
       <hr className='divider'/>
       <div className='image'>IMAGE</div>
       <hr className='divider'/>
       <div className='description'> insert desc</div>
       <button className='menu-item-button'>Add to order </button>
   </div>
  );
};
