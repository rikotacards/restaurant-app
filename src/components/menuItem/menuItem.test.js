import React from 'react'; 
import { MenuItem } from './MenuItem';
import renderer from 'react-test-renderer';

describe('Render MenuItem', () => {
    it('should only render div item with image and name', () => {
        const testDataEmpty = {};
    const tree = renderer
      .create(<MenuItem singleItem={testDataEmpty}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
    }),

    it('should show one line item', () => {
        const testData = {name: 'Morning glory', price:50};
        const tree = renderer
          .create(<MenuItem singleItem={testData}/>)
          .toJSON();
        expect(tree).toMatchSnapshot();
      });
})



 
  