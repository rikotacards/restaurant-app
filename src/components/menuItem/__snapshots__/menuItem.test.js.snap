// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MenuItem should only render div item with image and name 1`] = `
<div
  className="card"
>
  <div
    className="image"
  >
    IMAGE
  </div>
  <div
    className="title"
  />
  <div
    className="item-price"
  />
  <div
    className="description"
  />
  <button
    className="menu-item-button"
  >
    Add to order
  </button>
</div>
`;

exports[`Render MenuItem should show one line item 1`] = `
<div
  className="card"
>
  <div
    className="image"
  >
    IMAGE
  </div>
  <div
    className="title"
  >
    Morning glory
  </div>
  <div
    className="item-price"
  >
    50
  </div>
  <div
    className="description"
  />
  <button
    className="menu-item-button"
  >
    Add to order
  </button>
</div>
`;
